resources/F1_Drivers.csv

```diff
@@ -0,0 +1,22 @@
+DRIVER_ID,DRIVER_NAME,NATIONALITY,TEAM,PTS
+1,Max Verstappen,NED,RED BULL RACING HONDA,226.5
+2,Lewis Hamilton,GBR,MERCEDES,221.5
+3,Valtteri Bottas,FIN,MERCEDES,141
+4,Lando Norris,GBR,MCLAREN MERCEDES,132
+5,Sergio Perez,MEX,RED BULL RACING HONDA,118
+6,Charles Leclerc,MON,FERRARI,104
+7,Carlos Sainz,ESP,FERRARI,97.5
+8,Daniel Ricciardo,AUS,MCLAREN MERCEDES,83
+9,Pierre Gasly,FRA,ALPHATAURI HONDA,66
+10,Fernando Alonso,ESP,ALPINE RENAULT,50
+11,Esteban Ocon,FRA,ALPINE RENAULT,45
+12,Sebastian Vettel,GER,ASTON MARTIN MERCEDES,35
+13,Lance Stroll,CAN,ASTON MARTIN MERCEDES,24
+14,Yuki Tsunoda,JPN,ALPHATAURI HONDA,18
+15,George Russell,GBR,WILLIAMS MERCEDES,15
+16,Nicholas Latifi,CAN,WILLIAMS MERCEDES,7
+17,Kimi Räikkönen,FIN,ALFA ROMEO RACING FERRARI,2
+18,Antonio Giovinazzi,ITA,ALFA ROMEO RACING FERRARI,1
+19,Mick Schumacher,GER,HAAS FERRARI,0
+20,Robert Kubica,POL,ALFA ROMEO RACING FERRARI,0
+21,Nikita Mazepin,RAF,HAAS FERRARI,0
```

Comment:
The CSV files should have an empty line at the end because if your app adds a new registry, this will be added in this same row.